logistics/scripts/modules/utility/cross-platform.psm1

```diff
@@ -0,0 +1,19 @@
+function Get-IsWindows {
+    <#
+    .SYNOPSIS
+        Checks to see if the current machine is a Windows machine.
+    
+    .EXAMPLE
+        Get-IsWindows
+
+        returns $True
+    #>
+
+    if ($null -eq $IsWindows) {
+        $IsWindows = $true
```

Comment:
This looks wrong. I know that it is actually right though. Please add comments explaining why this is right.

Reply:
add some comments explaining